packages/Textarea/src/Textarea.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import textareaStyles from "./Textarea.styles";
+
+const propTypes = {
+  a11yText: PropTypes.string,
+  canExpand: PropTypes.bool,
+  className: PropTypes.string,
+  hasError: PropTypes.bool,
+  inputRef: PropTypes.func,
+  isDisabled: PropTypes.bool,
+  isReadOnly: PropTypes.bool,
+  maxHeight: PropTypes.string,
+  size: PropTypes.oneOf(ShirtSizes.DEFAULT),
+  value: PropTypes.string.isRequired,
+};
+
+const defaultProps = {
+  a11yText: null,
+  canExpand: true,
+  className: null,
+  hasError: false,
+  inputRef: () => {},
+  isDisabled: false,
+  isReadOnly: false,
+  maxHeight: "300px",
+  size: "medium",
+};
+
+class Textarea extends React.Component {
+  componentDidMount() {
+    if (this.props.canExpand) {
+      this.resize();
+      window.addEventListener("resize", this.resize);
+    }
+  }
+
+  componentDidUpdate(prevProps) {
+    if (this.props.canExpand) {
+      if (prevProps.canExpand && this.props.value !== prevProps.value) {
```

Comment:
I don't think we care about `prevProps.canExpand`?
So these two lines can be collapsed to:
`if (this.props.canExpand && this.props.value !== prevProps.value) {`

Reply:
@alexlaj, in the `acl-ui` version there was an event listener for window resizing... we didn't think it was still necessary though.  Do you remember why we had it originally?
https://github.com/acl-services/acl-ui/blob/master/src/components/__beta__/Textarea/Textarea.js#L11

Reply:
I asked him already, its needed when window is resized horizontally